services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -778,6 +781,25 @@ public void installDropFlow(final DatapathId dpid) throws SwitchOperationExcepti
         }
     }
 
+    void installDropLoopRule(DatapathId dpid) throws SwitchOperationException {
+        IOFSwitch sw = lookupSwitch(dpid);
+        OFFactory ofFactory = sw.getOFFactory();
+
+        if (ofFactory.getVersion() == OF_12) {
+            logger.debug("Skip installation of drop loop rule for switch {}", dpid);
+        } else {
+            Builder builder = ofFactory.buildMatch();
+            builder.setExact(MatchField.ETH_DST, MacAddress.of(VERIFICATION_BCAST_PACKET_DST));
+            builder.setExact(MatchField.ETH_SRC, MacAddress.of(dpid));
```

Comment:
You can use same technique (`MacAddress.of(dpid)`) into PathVerificationService.

(minor)